static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -637,11 +660,31 @@ function GroupDetailsContent({
   const {openMergedIssuesDrawer} = useMergedIssuesDrawer({group, project});
   const {openIssueActivityDrawer} = useIssueActivityDrawer({group, project});
   const {isDrawerOpen} = useDrawer();
+  const location = useLocation();
+  const navigate = useNavigate();
 
   const {currentTab, baseUrl} = useGroupDetailsRoute();
 
   const hasStreamlinedUI = useHasStreamlinedUI();
 
+  const statsPeriod = useGetStatsPeriod(group);
+
+  useEffect(() => {
+    if (hasStreamlinedUI) {
+      navigate(
```

Comment:
Do you want to do this even if a statsPeriod already exists? Seems like you would want to be able to link to a specific statsPeriod